### ISSUE TITLE
GrayScott.jl in simulations hosted on a different repo

### ISSUE THREAD

**Issue by williamfgc** [ornladios/ADIOS2#3907]
**Describe the bug**
Not a bug, but mostly to report that the current GrayScott.jl under simulations is outdated. 
The updated version can be found at [JuliaORNL/GrayScott.jl](https://github.com/JuliaORNL/GrayScott.jl)

Asking the experts, I talked to @pnorbert and we are wondering what would be the best way to move forward (e.g. subproject, just a link in README, etc.) as it would add overhead to this repo to handle all the Julia boilerplate. Thanks!

**Comment by vicentebolea** [comment]
cc @spyridon97

**Comment by spyridon97** [comment]
Is there a reason the julia one lives in each own repository?

**Comment by williamfgc** [comment]
> Is there a reason the julia one lives in each own repository?

Good question, the focus of GrayScott.jl more on Julia tutorials and the reference paper, it's not such much ADIOS2 centric (in this case ADIOS2.jl) unlike the C++ /Python versions.

**Comment by spyridon97** [comment]
@pnorbert would you like to remove the julia example from the main repository?

**Comment by pnorbert** [comment]
My opinion is to remove the example and have a Julia directory with a single README.me to point to the Julia tutorials. 
Another option would be to sub to that example if someone insists.

**Comment by williamfgc** [comment]
yes, it's more to bring awareness that this example and the Julia ADIOS2.jl also exists if someone looks at this directory not so much to add all the Julia boilerplate (packaging, tests, etc.) here.
